docs/management/managing-indices.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[managing-indices]]
+== Managing Indices
+
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+The index management UI enables you to update your Elasticsearch indices
+either individually or in bulk. To open the UI, go to *Management* and click
+*Index Management*.
```

Comment:
The term "update" is rather nebulous--I'd be inclined to provide more information about the types of operations you can perform up front. Maybe:

...enables you view index settings, mappings, and statistics and perform management operations such as refreshing, flushing, clearing the cache, merging segments, and closing or deleting indices. It provides a convenient way to perform bulk operations on multiple indices.

Reply:
I like this edit.